### ISSUE TITLE
Chunk方法的问题：model/clip_model 第 314 行

### ISSUE THREAD

**Issue by 2Elian** [WentaoTan/MLLM4Text-ReID#12]
我注意到，您在数据处理中(datasets/bases第241和243行)，将相同的capion传入给tokenize，我知道这是因为在第242行随机掩码的时候会把caption_tokens也改变，所以才有了第243行。然后您在model/build第94行：mix_ids = torch.cat([caption_ids,ori_caption_ids],dim=0)在batch维度进行了合并。然后用CLIP获取特征，但在model/clip_model的第314行中，mlm_token, ori_token = mix_token.chunk(2,dim=1)这个切分为什么要在第1个维度进行呢？不是应该继续在batch维度上做么。如果我哪里理解的不对，还请您指正。谢谢作者做出的伟大作品。

**Comment by WentaoTan** [comment]
谢谢！您过誉了哈小打小闹小打小闹。
您请看同样的clip_model.py的353-355行
```
x = x.permute(1, 0, 2)  # NLD -> LND
x = self.transformer(x,modal)
x = x.permute(1, 0, 2)  # LND -> NLD
```
在transformer中，dim=1是batch size dimension，于是对dim=1做chunk

**Comment by 2Elian** [comment]
> 谢谢！您曾评价过哈小打小闹小打小闹。 您请看同样的clip_model.py的353-355行
> 
> ```
> x = x.permute(1, 0, 2)  # NLD -> LND
> x = self.transformer(x,modal)
> x = x.permute(1, 0, 2)  # LND -> NLD
> ```
> 
> 在transformer中，dim=1是batch size维度，所以对dim=1做chunk

哈哈哈哈，抱歉，忘了后面有这个操作了！回复的太及时了！非常感谢您，期待您后续更好的工作~
